components/payments.jsx

```diff
@@ -1,15 +1,16 @@
 import { Grid, Box, Card, IconButton, Paragraph, Text } from '@theme-ui/components';
-import { formatPrice } from 'lib/utils/text';
 import { format } from 'date-fns';
 import { FiCheckCircle, FiArrowDownCircle } from 'react-icons/fi';
 
-export const PaymentItemCard = ({ payment: { created, invoice, currency, amount } }) => {
+export const PaymentItemCard = ({ payment: { created, invoice, amount } }) => {
+  //Add an extra decimal place
+  amount = (Math.round(amount * 100) / 100).toFixed(2);
```

Comment:
```suggestion
  amount = (Number(amount)).toFixed(2);
```

I think that will work the same